externalAPI/tradeshiftAPI/supplierAPI_test.go

```diff
@@ -0,0 +1,31 @@
+package tradeshiftAPI
+
+import (
+	"testing"
+)
+
+func Test_buildAdvancedSearchValue(t *testing.T) {
+	type args struct {
+		name string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "positive scenario: should be return string-json with escaped \"name\" as json value",
```

Comment:
This description hardly reflects actual behavior.

Reply:
changed